test/ngAnimate/animateSpec.js

```diff
@@ -130,6 +130,24 @@ describe("animations", function() {
       };
     }));
 
+    it("should not alter the provided options input in any way throughout the animation", inject(function($animate, $rootElement, $rootScope) {
```

Comment:
If you remove the patch, this is the only test that fails. Shouldn't all tests fail if the patch is removed?